ktor-http/common/src/io/ktor/http/Query.kt

```diff
@@ -94,3 +94,23 @@ private fun trimStart(start: Int, end: Int, query: CharSequence): Int {
     while (spaceIndex < end && query[spaceIndex].isWhitespace()) spaceIndex++
     return spaceIndex
 }
+
+/**
+ * Converts parameters to query parameters by fixing the [Parameters.get] method
+ * to make it return an empty string for the parameters without value (e.g., `?empty`)
+ */
+public fun Parameters.withEmptyStringForValuelessKeys(): Parameters {
```

Comment:
Should we mark it with `@InternalAPI`?

Reply:
I would also make a fast path - check if there are keys without parameters to avoid object creation